pkg/search/proxy/controller.go

```diff
@@ -97,13 +103,14 @@ func NewController(option NewControllerOption) (*Controller, error) {
 	proxy := pluginruntime.NewFramework(allPlugins)
 
 	ctl := &Controller{
-		restMapper:           option.RestMapper,
-		negotiatedSerializer: scheme.Codecs.WithoutConversion(),
-		secretLister:         secretLister,
-		clusterLister:        clusterLister,
-		registryLister:       option.KarmadaFactory.Search().V1alpha1().ResourceRegistries().Lister(),
-		store:                multiClusterStore,
-		proxy:                proxy,
+		restMapper:                   option.RestMapper,
+		negotiatedSerializer:         scheme.Codecs.WithoutConversion(),
+		secretLister:                 secretLister,
+		clusterLister:                clusterLister,
+		registryLister:               option.KarmadaFactory.Search().V1alpha1().ResourceRegistries().Lister(),
+		store:                        multiClusterStore,
+		storageInitializationTimeout: option.StorageInitializationTimeout,
```

Comment:
I can't see the option in binary.

Reply:
This option:
https://github.com/kubernetes/kubernetes/blob/a5772bd42593f6492f5169eef49bc9884f95abba/staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go#L360

Reply:
added